schema/config-linux.json

```diff
@@ -97,11 +97,11 @@
                         "properties": {
                             "blkioWeight": {
                                 "id": "https://opencontainers.org/schema/bundle/linux/resources/blockIO/blkioWeight",
-                                "$ref": "defs-linux.json#/definitions/blkioWeightPointer"
+                                "$ref": "defs-linux.json#/definitions/blkioWeight"
```

Comment:
I have modified `blkioWeight` to `uint16Pointer` as it is typed in spec-go, so `blkioWeightPointer` is not needed any more